docs/Tutorials/demo/budget_with_farming.md

```diff
@@ -103,28 +103,28 @@ $BINARY start
 
 ### Step 3. Send a governance proposal to create a budget plan 
 
-Let's create `budget-proposal.json` file and copy the below JSON into a file. Depending on what budget plan you plan to create, you can customize the following values of the fields for your need. 
+Create the `budget-proposal.json` file and copy the following JSON contents into the file. Depending on what budget plan you create, you can customize values of the fields. 
 
-In this demo, we plan to create a budget plan that distributes partial amount of coins from the [FeeCollector module account](https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/keys.go#L15) which collects gas fees and ATOM inflation in Cosmos Hub. This will be used for Gravity DEX farming plan to `GravityDEXFarmingBudget` account. 
+In this demo, you create a budget plan that distributes partial amount of coins from the [FeeCollector module account](https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/keys.go#L15) that collects gas fees and ATOM inflation in Cosmos Hub. This budget plan will be used for Gravity DEX farming plan to `GravityDEXFarmingBudget` account. 
 
-`GravityDEXFarmingBudget` account is created using the following code snippet.
+The `GravityDEXFarmingBudget` account is created using the following code snippet.
 
 ```go
 // cosmos1228ryjucdpdv3t87rxle0ew76a56ulvnfst0hq0sscd3nafgjpqqkcxcky 
 sdk.AccAddress(address.Module("farming", []byte("GravityDEXFarmingBudget")))
 ```
 
-Explanation about each field in the JSON file
+where the fields in the JSON file are:
 
-- `name`: is the name of the budget plan used for display
-- `description`: is the description of the budget plan used for display
-- `rate`: is the distributing amount by ratio of the total budget source
-- `budget_source_address`: is the address where the source of budget comes from
-- `collection_address`: is the address that collects budget from the budget source address
-- `start_time`: is start time of the budget plan
-- `end_time`: is end time of the budget plan
+- `name`: the name of the budget plan used for display
+- `description`: the description of the budget plan used for display
+- `rate`: the distributing amount by ratio of the total budget source
+- `budget_source_address`: the address where the source of budget comes from
```

Comment:
```suggestion
- `source_address`: the address where the source of budget comes from
```

Reply:
feel free to commit suggestions and make changes as you need!